app/gui/integration-test/dashboard/assetPanel.spec.ts

```diff
@@ -19,6 +19,11 @@ function locateAssetPanelDescription(page: Page) {
   return locateAssetPanel(page).getByTestId('asset-panel-description')
 }
 
+/** Find the contents of the Markdown editor within the given {@link Locator}. */
+function locateMarkdownContent(locator: Locator) {
+  return locator.locator('.cm-content')
```

Comment:
className selectors are forbidden as selectors in tests

Reply:
Why it is so? We are not shy of using class selector in our tests.

Reply:
Because class selectors aren't sustainable. Causing false-positives because of className change isn't a good practice overall

Reply:
Quick searching, here are an example of best practices from cypress website: https://docs.cypress.io/app/core-concepts/best-practices#How-It-Works

Reply:
This class is part of the public API of CodeMirror.